src/api/members.ts

```diff
@@ -10,3 +10,17 @@ export const listMembers = async (
   );
   return data;
 };
+
+export const listMembersDepaginated = async (
+  teamId: number,
+  page = 1
+): Promise<MemberRecord[]> => {
+  const { data } = await axiosInstance.get<PaginatedList<MemberRecord>>(
+    `/api/teams/${teamId}/members/?page=${page}`
+  );
+  if (data.next) {
+    return data.results.concat(await listMembersDepaginated(teamId, page + 1));
+  } else {
+    return data.results;
+  }
+};
```

Comment:
This is a clever use of recursion, but I wonder if it's better than just depaginating the member's list on the backend. Probably fine for now.